### ISSUE TITLE
Text editor won't open despite the existence of bmgres.arc

### ISSUE THREAD

**Issue by BupManMushroom** [LordNed/Winditor#56]
I am attempting to use Winditor's text editor, yet when I click on that option, the program claims that "bmgres.arc could not be found", even though that exact file is indeed within the game root. I have attempted this with multiple builds of Winditor and have met with the same issue for all of them.
![problem](https://github.com/LordNed/Winditor/assets/169518887/6d2281d0-26db-48f0-acb4-04a193bf63f8)

**Comment by LagoLunatic** [comment]
The "Game Root" folder is not `Extracted files/root-unchanged/res/Msg` as you have set. It's supposed to be the root directory that contains just two folders: `files` and `sys` and nothing else. Then Winditor will look for bmgres.arc at: `(game root)/files/res/Msg/bmgres.arc` as you can see it's trying to do in your screenshot.

It doesn't look like you have a `files` folder at all based on your screenshot, so you probably extracted the game's files using the wrong program. Winditor only supports folders extracted in Dolphin's directory structure, which you can get by extracting the ISO with Dolphin itself, or with [GCFT](https://github.com/LagoLunatic/GCFT).

I'm not sure what program created `root-unchanged` for you, but Winditor doesn't support it.

**Comment by BupManMushroom** [comment]
I extracted the game using a program called GCR (Gamecube Rebuilder), which extracts all the files and displays them as shown in the screenshot. I have narrowed the layout of the folders in file explorer down to two folders labeled "files" and "sys" as you recommended, and the text editor opened successfully!

If you're curious about the "root-unchanged" folder, I simply copied all the extracted files and put them in a 2nd folder I created myself where they would remain unedited, while I use the main "root" folder for modding with GCR.

Thank you for the suggestion and for your time.
